src/event-bridge/target-input.ts

```diff
@@ -170,18 +169,13 @@ export const synthesizeEventBridgeTargets = (
       };
     } else if (isBinaryExpr(expr)) {
       if (expr.op === "+") {
-        if (isStringType(expr.left) || isStringType(expr.right)) {
-          const val = `${exprToInternalLiteral(
-            expr.left
-          )}${exprToInternalLiteral(expr.right)}`;
-          return {
-            value: val,
-            type: "string",
-          };
-        }
-        throw Error(
-          "Addition operator is only supported to concatenate at least one string to another value."
-        );
+        const val = `${exprToInternalLiteral(expr.left)}${exprToInternalLiteral(
+          expr.right
+        )}`;
+        return {
+          value: val,
+          type: "string",
+        };
```

Comment:
would adding two numbers give the expected result?

Reply:
Are you suggesting I error? if `expr.left` or `expr.right` aren't strings?

Reply:
There is no way to know if this is two strings being concatenated unless one is a literal string type.

Reply:
Figured out what you mean. Updated it to require left or right be a StringLiteralExpr